modules/dbparser/tests/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
-add_unit_test(TARGET test_timer_wheel INCLUDES ${PATTERNDB_INCLUDE_DIR} DEPENDS patterndb)
-add_unit_test(TARGET test_patternize INCLUDES ${PATTERNDB_INCLUDE_DIR} DEPENDS patterndb syslogformat)
-add_unit_test(LIBTEST TARGET test_patterndb INCLUDES ${PATTERNDB_INCLUDE_DIR}
-  DEPENDS patterndb basicfuncs syslogformat)
-add_unit_test(TARGET test_radix INCLUDES ${PATTERNDB_INCLUDE_DIR} DEPENDS patterndb)
+add_unit_test(TARGET test_timer_wheel DEPENDS patterndb)
+add_unit_test(TARGET test_patternize DEPENDS patterndb syslogformat)
+add_unit_test(LIBTEST TARGET test_patterndb DEPENDS patterndb basicfuncs syslogformat)
+add_unit_test(TARGET test_radix DEPENDS patterndb)
 target_compile_options(test_radix PRIVATE "-Wno-error=pointer-sign")
-add_unit_test(LIBTEST TARGET test_parsers INCLUDES ${PATTERNDB_INCLUDE_DIR})
+# test_parsers includes a .c file
+add_unit_test(LIBTEST TARGET test_parsers INCLUDES "${CMAKE_CURRENT_SOURCE_DIR}/../")
```

Comment:
Nice catch. I'll check it.

Reply:
Oh okay.. so as I noted in line 6, this test includes a .c file instead of linking to dbparser and testing the public interface.

`../` looks weird, I'll do something about it.